ansible/roles/oso_host_monitoring/templates/monitoring-config.yml.j2

```diff
@@ -424,7 +413,7 @@ host_monitoring_cron:
 
 - name: "Check EBS stuck volume attachment state"
   minute: "*/2"
-  job: "ops-runner -f -s 15 -n cseeviss.ebs.stuck.state cron-send-ec2-ebs-volumes-in-stuck-state --region {{ osohm_region }} --stuck-after 120 --aws-creds-profile snapshotter --verbose"
+  job: "ops-runner -f -s 15 -n cseeviss.ebs.stuck.state cron-send-ec2-ebs-volumes-in-stuck-state --region {{ osohm_region }} --stuck-after 120 --aws-creds-profile snapshotter --verbose"- name: "Check EBS stuck volume attachment state"
```

Comment:
Ahh yes, this is the result of tower not responding while clicking around the screen, probably selected and copied stuff. Good catch.